dymaxionlabs/utils.py

```diff
@@ -55,7 +55,7 @@ class NotFoundError(Exception):
 
 def get_api_url():
     """Get current API URL from environment"""
-    return os.getenv("DYM_API_URL", "https://api.dymaxionlabs.com")
+    return os.getenv("DYM_API_URL", "https://api.dymaxionlabs.com/v1")
```

Comment:
I'm thinking it'll be better to have this env var point to the host/base URL of the API, and hardcode the version on the `request` method, like this:

```diff
+ API_VERSION = "v1"
```

```diff
def request(method,
            path,
            body=None,
            files=None,
            params={},
            headers={},
            binary=False,
            parse_response=True):
    """Makes an HTTP request to the API"""
    headers = {'Authorization': 'Api-Key {}'.format(get_api_key()), **headers}
    request_method = getattr(session, method)
-   url = urljoin(get_api_url(), path)
+   url = urljoin(get_api_url(), API_VERSION, path)
```

That way the user won't be able to change the API version of the SDK (it shouldn't be allowed as the SDK classes and functions are expected to work only with the latest version anyway).